frontend/public/components/cluster-settings/cluster-operator-start-guide.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import { Button } from 'patternfly-react';
+
+import { ExternalLink } from '../utils';
+
+const seenOperatorStartGuide: string = 'seenOperatorStartGuide';
+
+export class ClusterOperatorStartGuide extends React.Component<ClusterOperatorStartGuideProps, ClusterOperatorStartGuideState> {
```

Comment:
Prefer React hooks for new components. See #1456

Reply:
It would be better to make the existing `StartGuide` component reusable so it takes in a component and `localStorage` key.